hive-metastore/src/main/java/org/apache/iceberg/hive/HiveMetadataPreservingTableOperations.java

```diff
@@ -131,6 +150,7 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
       boolean tableExists = metaClients.run(client -> client.tableExists(database, tableName));
       if (tableExists) {
         tbl = metaClients.run(client -> client.getTable(database, tableName));
+        logTable(tbl);
```

Comment:
Can we print one more line before each `logTable` call which provides some context so that the log lines are clearer?
Something along the lines of `Fetched table from metastore` and `Updating table in metastore`?

Reply:
Done.